WORKSPACE.dev.bazel

```diff
@@ -21,6 +21,8 @@ load("//kotlin:repositories.bzl", "kotlin_repositories")
 
 kotlin_repositories()
 
+register_toolchains("//src/main/starlark/core/compile/...")
```

Comment:
Same here, nice having these be explicit instead of globbing an entire directory tree.